cmd/uplink/cmd/cp.go

```diff
@@ -111,7 +111,7 @@ func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress
 	}
 
 	opts.Volatile.RedundancyScheme = cfg.GetRedundancyScheme()
-	opts.Volatile.EncryptionParameters = cfg.GetEncryptionScheme().ToEncryptionParameters()
+	opts.Volatile.EncryptionScheme = cfg.GetEncryptionScheme()
```

Comment:
Why we are renaming `EncryptionParameters` into `EncryptionScheme`?

Reply:
Jeff requested it to be changed to scheme..
https://github.com/storj/storj/pull/2363#discussion_r299103370

Reply:
Not really, he requested changing variable `encryptionScheme` into `encryptionParameter` in `encryptionScheme := cfg.EncryptionParameters`

Reply:
updated